app/models/user.rb

```diff
@@ -110,9 +111,12 @@ def can_prefill_emis?
     beta_enabled?(uuid, FormProfile::EMIS_PREFILL_KEY)
   end
 
+  ID_CARD_ALLOWED_STATUSES = %w(V1 V3 V6).freeze
+
   def can_access_id_card?
```

Comment:
Not specifically aimed at this PR 😄  but the User model's getting pretty hefty. Once we have policy files for authorization all the `can_x?` logic can move there. Maybe move `ID_CARD_ALLOWED_STATUSES` so all the constants are at the top of the class.

Reply:
Agreed about the `can_xyz` methods getting crufty. 

Will move the constant.